include/swift/SyntaxParse/SyntaxTreeCreator.h

```diff
@@ -48,6 +48,8 @@ class SyntaxTreeCreator: public SyntaxParseActions {
   /// tree.
   SyntaxParsingCache *SyntaxCache;
 
+  llvm::BumpPtrAllocator ScratchAlloc;
```

Comment:
This is unused.

Reply:
Whoops. I’ll delete it in the next PR. Thanks!